test/Nesting.jl

```diff
@@ -56,13 +56,24 @@ using Test, Sandbox, Scratch, Pkg
                     #verbose = true,
                 )
 
+                cmd = `/bin/sh -c "julia --color=yes --project test/nested/nested_child.jl"`
                 with_executor(executor) do exe
-                    @test success(run(exe, config, `/bin/sh -c "julia --color=yes --project test/nested/nested_child.jl"`))
+                    @test success(run(exe, config, cmd))
                 end
                 @test isfile(joinpath(rw_dir, "single_nested.txt"))
                 @test isfile(joinpath(rw_dir, "double_nested.txt"))
                 @test String(read(joinpath(rw_dir, "single_nested.txt"))) == "aperture\n"
                 @test String(read(joinpath(rw_dir, "double_nested.txt"))) == "science\n"
+
+                if executor <: DockerExecutor
+                    for privileges in [:no_new_privileges, :unprivileged]
+                        with_executor(executor; privileges) do exe
+                            p = run(exe, config, cmd; wait = false)
```

Comment:
Does it not work to just test !success() here?

Reply:
I tried that and couldn't get it to work. It kept throwing an exception.

Let me make a commit to see if it works this time.

Reply:
Yeah take a look at CI on the new commit. It doesn't like it when I do `@test !success`.